neopixel.py

```diff
@@ -199,6 +199,30 @@ def set_pixel(self, pixel_num, rgb_w, how_bright=None):
         else:
             self.pixels[pixel_num] = pix_value
 
+    def get_pixelRGB(self, pixel_num):
+        """
+        Get red, green and blue value of pixel on position <pixel_num>
```

Comment:
This doc string should also mention white.

Reply:
Updated as suggested

Reply:
I changed the method name to 'getPixel' rather than try to include RGB in the name as it now returns either RGB or RGBW. This better matches with 'SetPixel' too.